addons/framework/statsTracking/fn_statsInitLocal.sqf

```diff
@@ -56,4 +58,16 @@ switch true do
 	default {
 		["inf_gi_roles", 0, (getPlayerUID player)] remoteExec ["CMF_fnc_updateStatArray", 2];
 	};
-};
\ No newline at end of file
+};
+
+[
+	{!alive player},
+	{
+		params ["_Player_UID"];
```

Comment:
Use camelcase for variables and stop using underscores. An underscore should only begin a variable in sqf:

`_playerUID`